packages/web-frontend/src/containers/mobile/Dashboard.js

```diff
@@ -0,0 +1,160 @@
+/**
```

Comment:
Mostly taken from the old `packages/web-frontend/src/components/mobile/Dashboard/index.js`, but 
- No longer expands/contracts, as there's no entity nav below so it might as well always be expanded
- Includes the org unit photo for any org unit type that has one (previously assumed only facilities would)